docker-compose.yml

```diff
@@ -12,9 +12,11 @@ services:
     environment:
       DB_NAME: wordpress
       DB_PASS: root
-      PLUGINS: akismet
+      PLUGINS: >-
+        academic-bloggers-toolkit,
```

Comment:
Do you feel strongly about that? I think it is more the role of the documentation. If you follow the getting started steps, you will end up with plugins you didn't ask for. I personally don't think it is a good thing since people might wonder if it is a requirement to the project.